lib/charms/opensearch/v0/opensearch_distro.py

```diff
@@ -158,12 +158,12 @@ def is_node_up(self) -> bool:
         except (OpenSearchHttpError, Exception):
             return False
 
-    def run_bin(self, bin_script_name: str, args: str = None):
+    def run_bin(self, bin_script_name: str, args: str = None, stdin: str = None) -> str:
```

Comment:
It is not very clear why do we need "stdin" on top of the args

Reply:
Improved docstring